beacon-chain/types/block.go

```diff
@@ -1,6 +1,27 @@
 package types
 
-import "github.com/ethereum/go-ethereum/common"
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Block defines a beacon chain core primitive.
+type Block struct {
```

Comment:
In the ECDC video, Vitalik and Justin mentioned there's no `header` and `body` for a beacon chain block. The whole thing is just a block with fields